Cython/Utility/ImportExport.c

```diff
@@ -87,65 +124,36 @@ static PyObject *__Pyx_ImportDottedModule_WalkParts(PyObject *module, PyObject *
     return module;
 }
 
-static PyObject *__Pyx__ImportDottedModule(PyObject *name, PyObject *parts_tuple) {
-    PyObject *imported_module;
-    PyObject *module = __Pyx_Import(name, NULL, 0);
+static PyObject *__Pyx__ImportDottedModule(PyObject *name, PyObject *parts_tuple, PyObject *qualname) {
+    PyObject *module = __Pyx_Import(name, NULL, 0, qualname);
     if (!parts_tuple || unlikely(!module))
         return module;
-
-    // Look up module in sys.modules, which is safer than the attribute lookups below.
-    imported_module = __Pyx__ImportDottedModule_Lookup(name);
-    if (likely(imported_module)) {
-        Py_DECREF(module);
-        return imported_module;
-    }
-    PyErr_Clear();
```

Comment:
I'm not completely sure if this is removable or not.

Reply:
I thought it is because `__Pyx_Import` which is called right above this include the lookup in `sys.modules` now.